test/screenshot/golden.json

```diff
@@ -432,11 +432,11 @@
     }
   },
   "spec/mdc-list/classes/checkbox.html": {
```

Comment:
Remove these screenshot updates. These are deleted in PR #5442

Reply:
Done